lib/methods/import.js

```diff
@@ -751,8 +751,8 @@ module.exports = self => {
         // recomputeAllDocReferences
         //
         // Perform any crops not found in the existing attachment
-        for (const crop of [ ...(attachment.crops || []), ...(existing.crops || []) ]) {
-          if (!existing.crops.find(c => JSON.stringify(c) === JSON.stringify(crop))) {
+        for (const crop of attachment.crops || []) {
```

Comment:
Unrelated: we're checking whether crops already exist, so by definition we don't need to check the list of existing crops to see if they are... existing